geonode/upload/datastore.py

```diff
@@ -69,23 +69,34 @@ def prepare_import(self, **kwargs):
         """
         return self.handler().prepare_import(self.files, self.execution_id, **kwargs)
 
-    def start_import(self, execution_id, **kwargs):
+    def start_import(self, execution_id, task_name, **kwargs):
         """
         call the resource handler object to perform the import phase
         """
-        return self.handler().import_resource(self.files, execution_id, **kwargs)
+        layer_names, _, _ = self.handler().import_resource(self.files, execution_id, **kwargs)
+
+        # Register the tasks_status with the created key alternates
+        orchestrator.register_task_status(execution_id, layer_names, task_name, status="RUNNING")
```

Comment:
i would suggest to keep this in a private function of the Datastore object. In case we need to make changes to the function at least will be in a single place